--- linux-4.2.3-orig/fs/dcache.c	2015-10-03 13:52:18.000000000 +0200
+++ linux-4.2.3/fs/dcache.c	2015-12-09 09:23:41.894912677 +0100
@@ -101,7 +101,10 @@
 
 static unsigned int d_hash_mask __read_mostly;
+unsigned int d_hash_shift __read_mostly;
+EXPORT_SYMBOL_GPL(d_hash_shift);
 
-static struct hlist_bl_head *dentry_hashtable __read_mostly;
+struct hlist_bl_head *dentry_hashtable __read_mostly;
+EXPORT_SYMBOL(dentry_hashtable);
 
 static inline struct hlist_bl_head *d_hash(const struct dentry *parent,
 					unsigned int hash)
--- linux-4.2.3-orig/include/linux/dcache.h	2015-10-03 13:52:18.000000000 +0200
+++ linux-4.2.3/include/linux/dcache.h	2015-12-09 09:11:17.726197903 +0100
@@ -584,4 +584,11 @@
 	return upper;
 }
 
+/**
+ * Export pour le TP 7
+ */
+
+extern struct hlist_bl_head *dentry_hashtable __read_mostly;
+extern unsigned int d_hash_shift __read_mostly;
+
 #endif	/* __LINUX_DCACHE_H */
